doc/api-hidden.rst

```diff
@@ -220,6 +221,7 @@
    Variable.cumprod
    Variable.cumsum
    Variable.equals
+   Variable.expand_dims
```

Comment:
Can we keep Variable unchanged?

Reply:
do you mean keep the `.set_dims()` on  `Variable`? if so, 👍🏽